### ISSUE TITLE
cannot find type `__gthread_once_t` in module `root::std`

### ISSUE THREAD

**Issue by ctaggart** [ctaggart/octh#17]
```
error[E0412]: cannot find type `__gthread_once_t` in module `root::std`
    --> src\lib.rs:3208:50
     |
3208 |             pub static mut locale_facet__S_once: root::std::__gthread_once_t;
     |                                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `root::std`
     |
     = help: possible candidate is found in another module, you can import it into scope:
               `use root::__gthread_once_t;`
```

**Comment by ctaggart** [comment]
Adding it here worked:
``` rust
    pub mod std {
        #[allow(unused_imports)]
        use self::super::super::root;
        use root::__gthread_once_t;
```

**Comment by ctaggart** [comment]
fixed by #22
